## try...catch문

try...catch문은 실행할 코드를 try 블록에 작성하고 예외(exception)가 발생(throw)할 경우 예외에 대한 처리를 지정합니다.

## 오류 유형

catch 블록을 통해 오류가 발생했을 때 오류 메시지에 따른 내용을 이해하고 있으면 오류에 대한 빠른 조치가 가능해집니다.

## throw

try...catch문을 통해 발생하는 오류 외에도 프로그램 로직이 정상적으로 수행되기 위한 특정 조건을 만족하지 못했을 때 의도적으로 예외를 발생시킬 때 사용합니다.

자세한 내용은 https://jeeee712.tistory.com/6 에 정리해 놓았습니다.
